### PR TITLE
language for firefox reviewers

### PR DESCRIPTION
**Pull request by toddtarsi**
**Thanks for contributing to the Selenium IDE!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium-ide/blob/master/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [ ] I have read the [contributing](https://github.com/SeleniumHQ/selenium-ide/blob/master/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by toddtarsi** [comment]
I just want to note here that I'm biting the language pretty much verbatim from the Katalon recorder team. Does anyone know a good way to credit them in a message here or give attribution?

**Comment by toddtarsi** [comment]
I raised an issue with them and asked how they'd like attribution:

https://github.com/katalon-studio/katalon-recorder/issues/108

They're a good bunch. That feels right.

**Comment by toddtarsi** [comment]
Okay, I changed the language slightly to be unique and referenced the Katalon team in the commit log. That's attribution enough I feel. This should get merged asap.

File: README.md [hunk 1]
```diff
@@ -63,4 +63,4 @@ You can also find us on on the [#selenium](irc://freenode.net/selenium) IRC chan
 
 ## For Mozilla Reviewers
 
-The use of CSP and eval is necessary for a record & playback tool (including inject JavaScript PER user commands). There is no other way to do this so please don't disable our add on again.
+Using CSP and eval is necessary for a browser-based tool that implements the webdriver API for recording and playback, since part of that spec allows users to inject javascript commands and perform ad-hoc evaluations of the results. Please don't disable us for this.
```
Comment (harsha509) [inline README.md:66]: Can we modify webdriver and javascript -> `WebDriver` API and `JavaScript`

**Comment by toddtarsi** [comment]
Closing this PR and redirecting discussion to https://github.com/SeleniumHQ/selenium-ide/pull/1310, since this changes the language and has a necessary version increment

File: README.md (hunk 1 above)
Comment (toddtarsi) [reply README.md:66]: Done in the other PR, good call!

**Pull request closed without merging**